flexibleengine/resource_flexibleengine_as_policy_v1_test.go

```diff
@@ -24,6 +24,13 @@ func TestAccASV1Policy_basic(t *testing.T) {
 					testAccCheckASV1PolicyExists("flexibleengine_as_policy_v1.hth_as_policy", &asPolicy),
 				),
 			},
+			{
+				ResourceName:      "flexibleengine_as_policy_v1.hth_as_policy",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
```

Comment:
if we set `scaling_group_id` in [resourceASPolicyRead](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/blob/master/flexibleengine/resource_flexibleengine_as_policy_v1.go#L238)
```go
d.Set("scaling_group_id", asPolicy.ID)
```
then **ImportStateVerifyIgnore** can be deleted.

Reply:
`d.Set` added and `ImportStateVerifyIgnore`removed

```
make testacc TEST='./flexibleengine' TESTARGS='-run TestAccASV1Policy_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./flexibleengine -v -run TestAccASV1Policy_basic -timeout 720m
=== RUN   TestAccASV1Policy_basic
--- PASS: TestAccASV1Policy_basic (28.35s)
PASS
ok      github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/flexibleengine 28.907s
```